src/main/java/core/basesyntax/FruitTransaction.java

```diff
@@ -0,0 +1,71 @@
+package core.basesyntax;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.Fruit;
+import core.basesyntax.service.FruitTransactionService;
+import core.basesyntax.service.ReaderService;
+import core.basesyntax.service.WriterService;
+import core.basesyntax.service.impl.ReaderServiceImpl;
+import core.basesyntax.service.impl.WriterServiceImpl;
+import core.basesyntax.strategy.FruitsTransactionStrategy;
+import java.util.ArrayList;
+import java.util.List;
+
+public class FruitTransaction implements FruitTransactionService {
+    private static final ReaderService readerService = new ReaderServiceImpl();
+    private static final WriterService writerService = new WriterServiceImpl();
```

Comment:
It should be inside methods you don`t need this fields

Reply:
++, changed